gk/main.c

```diff
@@ -573,6 +639,12 @@ gk_proc(void *arg)
 
 	gk_conf_hold(gk_conf);
 
+	if (block_idx == 0) {
+		ret = gk_hold_arp_and_nd(gk_conf, lcore);
```

Comment:
Notice that the LPM table will change dynamically, so it's better to take care of the LLS communication while editing the LPM Table.